tweetapi/views.py

```diff
@@ -13,27 +14,31 @@ class UserView(generics.ListCreateAPIView):
     queryset = User.objects.all()
     serializer_class = UserSerializer
 
-class ProfileView(generics.ListCreateAPIView):
+class ProfileView(generics.ListCreateAPIView,TwitterUserPermission):
```

Comment:
views don't inherit from permission right?

Reply:
no sir it does not inherit